eras/allegra/impl/CHANGELOG.md

```diff
@@ -13,6 +15,7 @@
 
 ## 1.3.0.0
 
+* Add `ToExpr` instance for `AllegraUtxoEvent`
```

Comment:
Probably should go [below](https://github.com/IntersectMBO/cardano-ledger/pull/4169/files#diff-15549b26d4a93e42ea4073cfa3b2436e03b75f0e9c9e7b97f5e5526964d612d1R26) under "testlib"